test/cpp/modules/manual/BaTests.cpp

```diff
@@ -19,7 +19,7 @@ TEST(BaTests, TestProcess)
 #else
     ModuleLoader moduleLoader("../../../../src/cpp/modules/manual/Manual.dll");
```

Comment:
Ultimately this to not be hard coded I assume?  But another PR?

Reply:
The plan is to have (in near future) a parametrized test suite with paths to all dlls as a set of parameters. So that all the ITest<I, O> implementations could be tested simultaneously. But yes, in another PR.